proto/poktroll/application/tx.proto

```diff
@@ -103,11 +103,12 @@ message MsgUpdateParam {
 
   string name = 2;
   oneof asType    {
-    cosmos.base.v1beta1.Coin as_coin = 3;
+    int64 as_int64 = 3;
```

Comment:
How was this working before? We should have had the `as_int64` way before right?

Reply:
The application module already had the `MaxDelegatedGateways` param but did not have the `MsgUpdateParam` message, so we're catching up a bit here.